Didn't find class "com.adjust.sdk.Adjust" on Android

I'm having an issue with analytics-react-native-adjust:0.0.1-beta.3. The build runs fine on Android but upon launch the app crashes with the error below. The same error happens for the Amplitude module. I've added these modules using `yarn add` and used `yarn react-native link` after that. This same package/integration works well on iOS. Are we supposed to setup Adjust and Amplitude SDKs manually on Android? (this doesn't seem to be necessary on iOS)

```
java.lang.NoClassDefFoundError: Failed resolution of: Lcom/adjust/sdk/Adjust;
        at com.segment.analytics.android.integrations.adjust.AdjustIntegration.<init>(AdjustIntegration.java:52)
        at com.segment.analytics.android.integrations.adjust.AdjustIntegration$1.create(AdjustIntegration.java:39)
        at com.segment.analytics.Analytics.performInitializeIntegrations(Analytics.java:1460)
        at com.segment.analytics.Analytics$2$1.run(Analytics.java:279)
        at android.os.Handler.handleCallback(Handler.java:790)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:440)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
     Caused by: java.lang.ClassNotFoundException: Didn't find class "com.adjust.sdk.Adjust" on path: DexPathList[[zip file "/data/app/com.hofr-y4tWHkTk7RwzIfMT6CNW7A==/base.apk"],nativeLibraryDirectories=[/data/app/com.hofr-y4tWHkTk7RwzIfMT6CNW7A==/lib/arm, /data/app/com.hofr-y4tWHkTk7RwzIfMT6CNW7A==/base.apk!/lib/armeabi-v7a, /system/lib, /system/vendor/lib]]
```

--- a/packages/integrations/template/android/build.gradle
+++ b/packages/integrations/template/android/build.gradle
@@ -41,7 +41,8 @@ repositories {
 
 
 dependencies {
-    implementation '{{{dependency}}}'
-
-    compileOnly project(":@segment_analytics-react-native")
+    api project(":@segment_analytics-react-native")
+    api('{{{dependency}}}') {
+        transitive = true
+    }
 }

